examples/jsm/nodes/core/NodeFrame.js

```diff
@@ -34,6 +72,17 @@ class NodeFrame {
 
 			}
 
+		} else if ( updateType === NodeUpdateType.RENDER ) {
+
+			if ( this.rendererMap.get( node ) !== this.renderId || this.updateMap.get( node ) !== this.frameId ) {
```

Comment:
Why do we need a separate map? Maybe we can use one (and maybe rename it to `frameMap`)?

Reply:
`renderMap` would not be a `frameMap`, we can have several renderings, that is several RTT in a single frame, but we can rename, `frameMap` seems like a better name